src/data/faq_de.json

```
@@ -352,10 +352,10 @@
                         ]
                     }, {
                         "active": true,
-                        "title": "[Google/Android]: Benachrichtigung zu möglicher Begegnung... - Wöchentliches Update",
+                        "title": "[Apple/iOS]: 'COVID-19-Kontaktprotokoll' - Wöchentliches Update",
                         "anchor": "notif_weekly_update",
                         "textblock": [
-                            "Das ist eine allgemeine Benachrichtigung, die nicht bedeutet, dass Risiko-Begegnungen erkannt worden sind. Das Exposure Notification Framework sendet diese Benachrichtigung. Sie sieht so ähnlich aus wie die weiteren Benachrichtigungen aus der Corona-Warn-App, zum Beispiel, wenn Sie Bluetooth deaktiviert haben. Sie können Ihren Risikostatus direkt in der Corona-Warn-App einsehen."
+                            "Das ist eine allgemeine Benachrichtigung, die nicht bedeutet, dass Risiko-Begegnungen erkannt worden sind. Das Exposure Notification Framework sendet diese Benachrichtigung und zeigt das Icon an. Sie können Ihren Risikostatus direkt in der Corona-Warn-App einsehen."
```

Nitpick: In the english translation this PR adds the word `current`, which would either translate to `momentanen` or `aktuellen`
```suggestion
                            "Das ist eine allgemeine Benachrichtigung, die nicht bedeutet, dass Risiko-Begegnungen erkannt worden sind. Das Exposure Notification Framework sendet diese Benachrichtigung und zeigt das Icon an. Sie können Ihren aktuellen Risikostatus direkt in der Corona-Warn-App einsehen."
```
```suggestion
                            "Das ist eine allgemeine Benachrichtigung, die nicht bedeutet, dass Risiko-Begegnungen erkannt worden sind. Das Exposure Notification Framework sendet diese Benachrichtigung und zeigt das Icon an. Sie können Ihren momentanen Risikostatus direkt in der Corona-Warn-App einsehen."
```